scene/main/viewport.cpp

```diff
@@ -3757,6 +3764,29 @@ float Viewport::get_texture_mipmap_bias() const {
 
 #endif // _3D_DISABLED
 
+void Viewport::_propogate_world_2d_changed(Node *p_node) {
+	if (p_node != this) {
+		if (!p_node->is_inside_tree()) { // may not have entered scene yet
```

Comment:
```suggestion
		if (!p_node->is_inside_tree()) { // May not have entered scene yet.
```

Reply:
Do you want me to remove the condition entirely then?